packages/react-components/react-datepicker-compat/src/components/CalendarDayGrid/CalendarDayGrid.tsx

```diff
@@ -147,8 +147,8 @@ export const CalendarDayGrid: React.FunctionComponent<CalendarDayGridProps> = pr
           week={weeks[0]}
           weekIndex={-1}
           rowClassName={classNames.firstTransitionWeek}
-          ariaRole="presentation"
-          ariaHidden={true}
```

Comment:
Shouldn't you also remove these props from `CalendarDayGrid.types.ts`?

Reply:
These are not in the types, so it's safe to remove them.